packages/bot-web-ui/src/assets/sass/quick-strategy.scss

```diff
@@ -20,6 +20,7 @@
         font-weight: normal;
         line-height: 1.43;
         color: var(--text-general);
+        padding: 0px 18px;
```

Comment:
Hmm, could you find another way to place the scrollbar outside of the container? Adding 18px padding everywhere will break the design that we have.

Reply:
@aaron-binary at first I did same and I feel it's not pretty and then I change it to these but I change it again and then we could make a decision about the position of scroll :)